src/vs/workbench/services/positronConsole/browser/classes/activityItemOutputHtml.ts

```diff
@@ -1,12 +1,20 @@
 /*---------------------------------------------------------------------------------------------
- *  Copyright (C) 2023 Posit Software, PBC. All rights reserved.
+ *  Copyright (C) 2023-2025 Posit Software, PBC. All rights reserved.
  *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { localize } from '../../../../../nls.js';
+import { ActivityItem } from './activityItem.js';
+
+/**
+ * Localized strings.
+ */
+const positronHTMLOutput = localize('positronHTMLOutput', "[HTML output]");
```

Comment:
My recollection is that most items that return HTML representations also return a text version under a different MIME key. So we _could_ plumb that down to the activity item so that it has something to feed into the clipboard. (Probably for a followup change.)